code/game/objects/items/weapons/other.dm

```diff
@@ -901,3 +901,25 @@
 	icon = 'icons/obj/lavaland.dmi'
 	icon_state = "condensedphlogiston"
 	w_class = ITEMSIZE_SMALL
+
+//Crafting components for plasma weapons.
+/obj/item/copper_coil
+	name = "copper wire coil"
+	desc = "A coil of copper wire."
+	icon = 'icons/obj/items_vr.dmi'
+	icon_state = "coppercoil"
+	w_class = ITEMSIZE_NORMAL
+
+/obj/item/plasma_rifle_casing
+	name = "casing (plasma rifle)"
+	desc = "This hammered plasteel casing has been shaped and cut into the upper frame of a plasma rifle."
+	icon = 'icons/obj/items_vr.dmi'
+	icon_state = "caseplasmarifle"
+	w_class = ITEMSIZE_NORMAL
+
+/obj/item/plasma_pistol_casing
+	name = "casing (plasma pistol)"
+	desc = "This hammered plasteel casing has been shaped and cut into the upper frame of a plasma pistol."
+	icon = 'icons/obj/items_vr.dmi'
+	icon_state = "caseplasmapistol"
+	w_class = ITEMSIZE_NORMAL
```

Comment:
why do these exist if they're just used for like... 1 recipe each, 2 for the coil? just add the copper/plasteel costs to the recipes and increase the time to craft on both